pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[project]
+name = "TF2.4_IVIM-MRI-CodeCollection"
+version = "0.1.0"
+description = "IVIM MRI signal simulation and DICOM/NIfTI processing tools"
+authors = [{ name="Jackson Hardee", email="jphardee@gmail.com" }]
```

Comment:
Should probably be OSIPI? Might be good to get someone else's feedback on this too.